### PR TITLE
fature/header

### PR DESCRIPTION
**Pull request by PolztiukVolodymyr**
add header nav

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,6 +12,18 @@
     />
   </head>
   <body>
+    <header>
+      <div>
+        <nav>
+          
+            <li>tel</li>
```
Comment (TarasRudko) [inline src/index.html:19]: no \<ul\> tag

File: src/index.html [hunk 2]
```diff
@@ -12,6 +12,18 @@
...
+            <li>email</li>
+            <li>adres</li>
```
Comment (TarasRudko) [inline src/index.html]: typo, should be: 
address

**Review by TarasRudko — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (PolztiukVolodymyr) [reply src/index.html]: fixed

File: src/index.html (hunk 1 above)
Comment (PolztiukVolodymyr) [reply src/index.html:19]: fixed

**Review by TarasRudko — APPROVED** [review]

**Pull request merged**